x11-calc.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.mike632t.x11_calc</id>
+
+  <name>RPN Calculator</name>
+  <summary>An HP calculator emulator</summary>
+
+  <developer id="mike632t.wordpress.com">
+  <name>mike632t</name>
+  </developer>
+
+  <metadata_license>CC-BY-4.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <description>
+      <p>Available emulated models are:</p>
+      <ul>
+          <li>Classic serie (HP 35, HP 80, HP 45, HP 70)</li>
+          <li>2nd Generation / Woodstock serie (HP 21, HP 22, HP 25, HP 25c, HP 27, HP 29c, HP 67)</li>
+          <li>3rd Generation / Spice serie (HP 31e, HP 32e, HP 33e, HP 33c, HP 34c, HP 37e, HP 38c, HP 38e)</li>
+          <li>Voyager serie (HP 10c, HP 11c, HP 12c, HP 15c, HP 16c)</li>
+      </ul>
+    <p>Default model is 25c. Right-click on icon to set preferred model, or run <code>flatpak run io.github.mike632t.x11_calc --setup</code></p>
+  </description>
+
+  <launchable type="desktop-id">io.github.mike632t.x11_calc.desktop</launchable>
+  <content_rating type="oars-1.1" />
+
+  <url type="homepage">https://github.com/mike632t/x11-calc</url>
+  <url type="faq">https://www.hpmuseum.org</url>
+  <url type="bugtracker">https://github.com/mike632t/x11-calc/issues</url>
+  <url type="vcs-browser">https://github.com/mike632t/x11-calc</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/mike632t/x11-calc/stable/img/x11-calc-25.png</image>
+      <caption>HP 25c</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/mike632t/x11-calc/stable/img/x11-calc-11c.png</image>
```

Comment:
Images should be from a tag or a commit. https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots